problems/mean_en_post_PJFNK.i

```diff
@@ -1,3 +1,5 @@
+#Runs but Solve failed
```

Comment:
I was trying to do as you suggested and the input file is running, but the dt is staying around ~2e-10 (so the file is not done yet). The only difference between the two files are the potential BCs (mean_en has NeumannCircuitVoltageMoles_KV, while mean_en_PJFNK has CircuitDirichletPotential with the postprocessor) and their solve type (Newton and PJFNK). Maybe optimizing the PetsC options would help it run better, but I need to look into the options usually used for PJFNK.

Reply:
Okay, those differences make sense. So you do not get the "solve failed" that occurred in the previous iteration? If these small dt's were not expected behavior, then massaging the solve options may help. I would be wary of going down a rabbit hole here though. This could be work for a separate PR where we're cleaning up the problems directory.